cc33xx-target-scripts/suite/trixie/debian/copyright

```diff
@@ -0,0 +1,84 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cc33xx-target-scripts
+Upstream-Contact: Texas Instruments
+Source: https://git.ti.com/cgit/cc33xx-wlan/cc33xx-target-scripts/
+
+Files:     ap/ap_start.sh
+           ap/ap_stop.sh
+           conf/udhcpd.conf
+           conf/wpa_supplicant.conf
+           conf/wpa_supplicant2.conf
+           drv/am335x_sdio_rebind.sh
+           drv/am6x_sdio_rebind.sh
+           drv/antenna_selection.sh
+           drv/cc33xx_twt.sh
+           drv/diversity_enable.sh
+           drv/diversity_select_default_antenna.sh
+           drv/diversity_set_rssi_threshold.sh
+           drv/load_cc33xx.sh
+           drv/set_lsi.sh
+           drv/set_power_mode.sh
+           drv/unload_cc33xx.sh
+           sta/sta_add.sh
+           sta/sta_connect-ex-dhcp.sh
+           sta/sta_connect-ex.sh
+           sta/sta_connect.sh
+           sta/sta_remove.sh
+           sta/sta_start.sh
+           sta/sta_stop.sh
+Copyright: {2011} Texas Instruments Incorporated - http://www.ti.com/
+License:   TI-TFL
+ /*
+  * copyright(c) 1998 - 2024 texas instruments. all rights reserved.
+  * all rights reserved.
+  *
+  *
+  * redistribution and use in source and binary forms, with or without
+  * modification, are permitted provided that the following conditions
+  * are met:
+  *
+  *  * redistributions of source code must retain the above copyright
+  *    notice, this list of conditions and the following disclaimer.
+  *  * redistributions in binary form must reproduce the above copyright
+  *    notice, this list of conditions and the following disclaimer in
+  *    the documentation and/or other materials provided with the
+  *    distribution.
+  *  * neither the name texas instruments nor the names of its
+  *    contributors may be used to endorse or promote products derived
+  *    from this software without specific prior written permission.
+  *
+  * this software is provided by the copyright holders and contributors
+  * "as is" and any express or implied warranties, including, but not
+  * limited to, the implied warranties of merchantability and fitness for
+  * a particular purpose are disclaimed. in no event shall the copyright
+  * owner or contributors be liable for any direct, indirect, incidental,
+  * special, exemplary, or consequential damages (including, but not
+  * limited to, procurement of substitute goods or services; loss of use,
+  * data, or profits; or business interruption) however caused and on any
+  * theory of liability, whether in contract, strict liability, or tort
+  * (including negligence or otherwise) arising in any way out of the use
+  * of this software, even if advised of the possibility of such damage.
+  */
+ .
+
+Files:     conf/hostapd.conf
```

Comment:
Why is this one file a different license? Why does a config file have a license at all.. Could you see if the CC33xx folks can fix this up?

Reply:
I have asked them. Will update this once they get back.

Reply:
@glneo This has been fixed upstream, so I have removed hostapd.conf from the copyright.